osu.Game.Rulesets.Osu/Edit/Blueprints/Sliders/Components/PathControlPointVisualiser.cs

```diff
@@ -115,6 +116,9 @@ public void OnReleased(PlatformAction action)
         {
         }
 
+        public override bool ReceivePositionalInputAt(Vector2 screenSpacePos) =>
+            Pieces.Any(p => p.ReceivePositionalInputAt(screenSpacePos));
```

Comment:
Is this required? This override is what seems to specifically break tests (because of this the visualiser can't receive click events outside of any path piece, breaking deselection), and it seems to work fine without as far as I can tell.

Reply:
Without this, points that don't overlap the slider body will not be deletable. Easy way to test is making a slider with four points, where the third doesn't follow a straight line.

Reply:
Weird, I explicitly remember guessing it was for that and testing such a scenario - unless I screwed up, seemed to work.